rikai/public/js/custom.js

```diff
@@ -0,0 +1,568 @@
+//preloading for page
+$(window).on('load', function() { // makes sure the whole site is loaded 
+	var status = $('#status');
+	var preloader = $('#preloader');
+	var body = $('body');
+	status.fadeOut(); // will first fade out the loading animation 
+	preloader.delay(0).fadeOut('fast'); // will fade out the white DIV that covers the website. 
+	body.delay(0).css({'overflow':'visible'});
+	var vidDefer = document.getElementsByTagName('iframe');
+	for (var i=0; i<vidDefer.length; i++) {
+		if(vidDefer[i].getAttribute('data-src')) {
+			vidDefer[i].setAttribute('src',vidDefer[i].getAttribute('data-src'));
+		} 
+	}
+})
+$(function(){
+	'use strict';
+	// js for dropdown menu
+	var windowWidth = $(window).width();
+	if(windowWidth > 1024){
+		var dropdown = $( '.dropdown');
+		dropdown.hover(
+	        function(){
+	            $(this).children('.dropdown-menu').fadeIn(300);
+	        },
+	        function(){
+	            $(this).children('.dropdown-menu').fadeOut(300);
+	        }
+	    );	   
+	}else{
+		var dropdownClick = $('.navbar a.dropdown-toggle');
+		dropdownClick.on('click', function(e) {
+			var $el = $(this);
+			var $parent = $(this).offsetParent(".dropdown-menu");
+			var $open = $('.nav li.open');
+			$(this).parent("li").toggleClass('open');
+
+			if(!$parent.parent().hasClass('nav')) {
+				$el.next().css({"top": $el[0].offsetTop, "left": $parent.outerWidth() - 4});
+			}
+			$open.not($(this).parents("li")).removeClass("open");
+			return false;
+		});
+	}
+	//js for nav icon 
+	var clickMenubtn = $('#nav-icon1');
```

Comment:
Codacy has a fix for the issue: [Strings must use doublequote.](https://app.codacy.com/gh/rikai-academy/book_system_01/pullRequest?prid=7834536)
```suggestion
	var clickMenubtn = $("#nav-icon1");
```